web/static/js/snake.js

```diff
@@ -2,10 +2,117 @@ import $ from "jquery";
 import Vue from "vue";
 import * as THREE from "three";
 import * as TWEEN from "tween.js";
+import { Howler, Howl } from 'howler';
 
 var SNAKE_HEAD_URL = "/images/snake_head_nerdy.png";
 var SNAKE_HEAD_SCALE = 1.5;
 var FOOD_URL = "/images/food.png"
+var AUDIO_URL = "/audio/"
+var AUDIO_PLAYLISTS_URL = AUDIO_URL + "playlists.json"
```

Comment:
in ES6 it's recommended that you use either `let` or `const` for variable declarations. Although brunch will save you here by wrapping all modules in a IIFE, in general this would create a global variable.